components/FAQ/molecules/QAndItem.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { IoChevronUpCircleOutline, IoChevronDownCircleOutline } from 'react-icons/io5';
+import { useState } from 'react';
+
+import { Green050, Green100, Grey050 } from 'components/styling/colors';
+import { DefaultParagraph, LargeParagraph } from 'components/atoms/typography.styles';
+
+const QuestionHeader = styled.div`
+  display: grid;
+  grid-template-columns: auto 28px;
+  column-gap: 28px;
+
+  @media screen and (min-width: 1200px) {
+    column-gap: 68px;
+  }
+`;
+
+const DividerLine = styled.div`
+  height: 1px;
+  background-color: ${Grey050};
+  margin: 40px 0 30px;
+
+  @media screen and (min-width: 1200px) {
+    margin-bottom: 40px;
+  }
+`;
+
+const DefaultParagraphWithTopPadding = styled(DefaultParagraph)`
+  padding-top: 20px;
+  
+  @media screen and (min-width: 1200px) {
+      padding-top: 16px;
+  }
+`;
+
+interface Props {
+  item: {
+    question: string;
+    answer: string;
+  };
+  isLastItem: boolean;
+}
+
+const QAndAItem = ({ item, isLastItem }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
I'd highly recommend animating this. Not much effort, and makes the UX a lot better.

I can recommend using this lib, should be pretty lightweight, and easy to use.
https://github.com/Stanko/react-animate-height